src/shared/ui/list/list.tsx

```diff
@@ -0,0 +1,44 @@
+// todo delete TextWithLink
+import classNames from 'classnames/bind';
+import { TextWithLink } from '../text-with-link';
+import { LinkList } from '@/widgets/required/required.types';
+
+import styles from './list.module.scss';
+
+export type ListProps = {
+  data: ListData;
+  marked?: boolean;
+};
+
+export type ListData = (string | LinkList)[] | [] | undefined;
+
+export const cx = classNames.bind(styles);
+
+export const List = ({ data, marked = true }: ListProps) => {
+  if (!data || !data.length) {
+    return <ul data-testid="list"></ul>;
+  }
+
+  return (
+    <ul
+      className={cx('list', { marked: marked === true })}
```

Comment:
```suggestion
      className={cx('list')}
```

No need to apply this class on the `ul` itself, because this will do nothing, because of how the styles is being written

<img width="478" alt="Screenshot 2024-08-14 at 22 25 08" src="https://github.com/user-attachments/assets/a5dd5eb3-b008-4164-9fe0-0933d968e2ae">



You also need to change the test too
<img width="838" alt="Screenshot 2024-08-14 at 22 27 12" src="https://github.com/user-attachments/assets/26e67bfa-1786-4c75-a69c-0f08c90d89a8">

Reply:
I've removed class `marked` from `listItems` and changed CSS-file